### PR TITLE
responsive layout + animations + fixed issues

### PR DESCRIPTION
**Pull request by Shitikyan**

### CODE REVIEW

File: quantum-be/src/telnyxService.js [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
 
-const telnyxApiKey = "YOUR_TELNYX_API_KEY";
-telnyx.init({ apiKey: telnyxApiKey });
+const telnyxApiKey =
+  "KEY0189DA432F175141D5E2CE9125F9482B_l6tLeXw4CQjzvCuHdBtAYB";
+const telnyxTool = telnyx(telnyxApiKey);
 
-const sendCongratulatoryEmail = async (userEmail) => {
+const sendCongratulatoryEmail = async (_, res) => {
+  console.log("????????????");
```
Comment (Shitikyan) [inline quantum-be/src/telnyxService.js]: delete

File: quantum/public/particles.js-master/.jshintrc
Comment (Shitikyan) [inline quantum/public/particles.js-master/.jshintrc:1]: why do you need this file? `dist` folder is generating during build, so it must be in `gitignore` file.

File: quantum/public/particles.js-master/.travis.yml
Comment (Shitikyan) [inline quantum/public/particles.js-master/.travis.yml:1]: why do you need this file?

File: quantum/public/particles.js-master/LICENSE
Comment (Shitikyan) [inline quantum/public/particles.js-master/LICENSE:1]: delete

File: quantum/public/particles.js-master/dist/particles.js
Comment (Shitikyan) [inline quantum/public/particles.js-master/dist/particles.js:1]: why do you need this file?

File: quantum/public/particles.js-master/dist/particles.min.js
Comment (Shitikyan) [inline quantum/public/particles.js-master/dist/particles.min.js:1]: what is this?

File: quantum/public/particles.js-master/gulpfile.js
Comment (Shitikyan) [inline quantum/public/particles.js-master/gulpfile.js:1]: check careful whole `particles.js-master` folder

File: quantum-be/src/Services/telnyxService.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+const sendCongratulatoryEmail = async (_, res) => {
+  console.log("????????????");
+  try {
+    const message = await telnyxTool.sendEmail.create({
+      from: "aiquantum15@gmail.com",
+      to: "aiquantum15@gmail.com",
+      text: "CONGRATULATIONS! YOUR ORDER IS ACCEPTED! Our operator will contact you shortly for confirmation.",
+    });
+    console.log(message);
```
Comment (SergeyG-Solicy) [inline quantum-be/src/Services/telnyxService.js:15]: delete

File: quantum/src/services/userService.ts [hunk 3]
```diff
@@ -1,21 +1,17 @@
...
-      `https://quantum.solicy.net/users/signup`,
-      body,
-      {
-        headers: {
-          "Content-Type": "application/json",
-          "Access-Control-Allow-Origin": "*",
-        },
-      }
-    );
+  public async signin(body: IUser): Promise<string | AxiosError> {
```
Comment (SergeyG-Solicy) [inline quantum/src/services/userService.ts:8]: must be `signup`

**Review by SergeyG-Solicy — CHANGES_REQUESTED** [review]

**Pull request merged**